packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -86,3 +87,20 @@ export function adaptWithProxy<From extends object, To extends object>(
 export function getArrayOwnKeys(length: number): string[] {
     return Object.getOwnPropertyNames(Array.from(Array(length)));
 }
+
+const pathDelimiter = "/";
+
+function stringifyPath(path: UpPath, childPath: ETreeNodePath): ETreeNodePath {
+    const fieldName = typeof path.parentField === "string" ? path.parentField : keyFromSymbol(path.parentField);
```

Comment:
This does not handle global and local fields with the same name correctly.

Reply:
also, if you do want to check which type a field key is, use isGlobalFieldKey instead of manually using typeof